### DISCUSSION TITLE
Order of the columns in the data download.

### DISCUSSION THREAD

**Discussion by ortnever** [visualize-admin/visualization-tool#752] [Q&A]
Dear Visualize-team,

Is the order of the columns in the "Download of full dataset" configured via Visualize?  

Thank you for your answer.

**Comment by ptbrowne** [comment]
Yes, the order of column is controlled by visualize. Right now, the only thing that is specified is that there are the dimensions and then measures.

**Comment by ortnever** [comment]
In this case it would be good to use the same order as defined for the preview table (shacl#order?). To be done occasionally if it's not a big thing.

**Reply by ptbrowne** [reply]
I created https://github.com/visualize-admin/visualization-tool/issues/753

**Reply by ortnever** [reply]
Thank you !

**Comment by ortnever** [comment]
I also asked myself how the sorting within the columns is defined. When we have a year column, for example, the years do not appear chronologically in the download. 
Does Visualize have an influence on this too?

**Comment by ortnever** [comment]
it's not urgent of course ;-)
